iroh-bytes/src/downloader/invariants.rs

```diff
@@ -73,15 +74,15 @@ impl<G: Getter<Connection = D::Connection>, D: Dialer> Service<G, D> {
         }
     }
 
-    /// Checks that the scheduled requests match the queue that handles their delays.
-    #[track_caller]
-    fn check_scheduled_requests_consistency(&self) {
-        assert_eq!(
-            self.scheduled_requests.len(),
-            self.scheduled_request_queue.len(),
-            "scheduled_request_queue and scheduled_requests are out of sync"
-        );
-    }
+    // /// Checks that the scheduled requests match the queue that handles their delays.
```

Comment:
What's with this? No longer needed?

Reply:
Ah right, the delay queue doesn't exist anymore. As noted in the description above, no retries atm because we first need to decide how they should work.

Left this in earlier to see if there's a similar assert to be made now. Will think through and either remove or replace

Reply:
Replaced with an actual check.